features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/VERSION.txt

```diff
@@ -1 +1 @@
-1d1faca481c3
+36e356a5
```

Comment:
This is only 8, not 12, characters long. The Cypress TF-M version should go in a new VERSION.txt file in the same folder as the binaries, rather than here. This is the version for all targets that are updated together, which PSOC 64 won't be for now.

Reply:
Done